src/qt/clientmodel.cpp

```diff
@@ -306,7 +297,10 @@ static void BlockTipChanged(ClientModel* clientmodel, SynchronizationState sync_
 void ClientModel::subscribeToCoreSignals()
 {
     // Connect signals to client
-    m_handler_show_progress = m_node.handleShowProgress(std::bind(ShowProgress, this, std::placeholders::_1, std::placeholders::_2));
+    m_handler_show_progress = m_node.handleShowProgress(
+        [this](const std::string& title, int progress, [[maybe_unused]] bool resume_possible) {
+            Q_EMIT showProgress(QString::fromStdString(title), progress);
```

Comment:
508e2dca5e91c1ff921f01d260fc62f629f1dc9e

This is fine because auto connection is used:
https://github.com/bitcoin-core/gui/blob/747cdf1d652d8587e9f2e3d4436c3ecdbf56d0a5/src/qt/bitcoingui.cpp#L593